tools/devops/automation/post-build-pipeline.yml

```diff
@@ -0,0 +1,61 @@
+# YAML pipeline for post build operations. 
+# This pipeline will trigger automatically after successful completion of the `build_packages` stage against the specified branches.
+
+trigger: none
+pr: none
+
+resources:
+  pipelines:
+  - pipeline: macios
+    source: xamarin-macios
+    trigger:
+      branches:
+      - main
+      - 6.0.*
```

Comment:
One thing I messed up on here: https://github.com/dotnet/maui/issues/598

The branches should be named `release/6.0.1xx-preview3` next time, such as:

https://github.com/dotnet/installer/tree/release/6.0.1xx-preview3

Should we make the trigger `release/*`? I can update xamarin-android for this, too.

Reply:
Updated